web/src/components/tasks/business-formation/billing/BillingStep.tsx

```diff
@@ -23,37 +24,43 @@ export const BillingStep = (): ReactElement => {
       >
         <div className="grid-row grid-gap-1">
           <div className="margin-top-2 tablet:grid-col-6">
-            <BusinessFormationTextField
-              label={Config.formation.fields.contactFirstName.label}
-              fieldName="contactFirstName"
-              errorBarType="MOBILE-ONLY"
-              required={true}
-              validationText={getFieldErrorLabel("contactFirstName")}
-            />
+            <FormationField fieldName="contactFirstName">
```

Comment:
Most of this PR is whitespace changes due to wrapping fields in this new `FormationField` component.